src/status_im/translations/fr.cljs

```diff
@@ -154,8 +147,18 @@
    :search-tags                           "Entrez vos mots clés ici"
    :popular-tags                          "Mots-clés populaires"
    :recent                                "Récent"
-   :no-statuses-discovered                "Aucun statut découvert"
    :no-statuses-found                     "Aucun statut trouvé"
+   :chat                                  "Chat"
+   :all                                   "Tout"
+   :public-chats                          "Chats public"
+   :soon                                  "Prochainement"
+   :public-chat-user-count                "{{count}} personne"
+   :dapps                                 "ÐApps"
+   :dapp-profile                          "profile ÐApp"
+   :no-statuses-discovered                "Aucun statut découvert"
+   :no-statuses-discovered-body           "Quand quelqu'un écrira un post\nvous le verrez ici."
+   :no-hashtags-discovered-title          "Aucun #hashtag trouvé"
+   :no-hashtags-discovered-body           "Quant un #hashtag deviens\npopulaire vous le verrez ici."
```

Comment:
`Quant` -> `Quand`
`deviens` -> `devient`

Reply:
yep, fixing it.